src/components/CampaignPage.js

```diff
@@ -155,8 +157,14 @@ class CampaignPage extends Component {
     handleDonateFiat = async () => {
         //TODO: check that this.state.donationAmount is larger than 0
         let cardKeyData = await getPCIPublicKey();
```

Comment:
So, this is getting called whether Circle is enabled or not, but if Circle API is not configured or we don't have access to it, this will fail. This statement should be moved inside `if(this.state.fiatPaymentProvider === 'circle'){`

Reply:
done